clients/fides-js/src/lib/initialize.ts

```diff
@@ -235,7 +235,9 @@ export const initialize = async ({
         `User location could not be obtained. Skipping overlay initialization.`
       );
       shouldInitOverlay = false;
-    } else if (!isPrivacyExperience(experience)) {
+      // An empty experience (e.g. {}) is expected when 1. pre-fetch is enabled, and 2. the location has no associated
+      // experience. We should not fetch experiences again in this case. We only fetch experiences if it's undefined.
+    } else if (!experience) {
```

Comment:
nit...
```suggestion
    } else if (!effectiveExperience) {
```

Reply:
TBH, this was clearer before using the utils method. I reworked that method to suit your purpose